src/app/tests/suites/certification/Test_TC_SC_6_1.yaml

```diff
@@ -363,74 +363,69 @@ tests:
           [1651471175.522467][4012:4012] CHIP:DMG: AccessControl: allowed
       disabled: true
 
-    - label:
-          "Step 7: DUT reads GroupKeyMap Attribute from the GroupKeyManagement
-          cluster from TH"
-      PICS: GRPKEY.C.A0000
+    - label: "Step 7: DUT sends KeySetRemove Command to TH"
+      PICS: GRPKEY.C.C03.Tx
       verification: |
-          ./chip-tool groupkeymanagement read group-key-map 1 0
+          ./chip-tool groupkeymanagement key-set-remove 42 1 0
```

Comment:
is this ever going to be automatable in yaml? If no, there's no point in fiddling further with this file as it will need to be re-written anyway.